Server/src/usercoordinator.erl

```diff
@@ -0,0 +1,69 @@
+-module(usercoordinator).
+-export([init/1]).
+
+%%Author: Tim Jonasson
+%%Version: 1
+
+%Initializes the usercoordinator
+init(Socket) -> 
+  inet:setopts(Socket, [{active, true}]),
+  loop(Socket, []).
+
+  
+loop(Socket, Diagrams) -> 
+  receive
+    {tcp, Port, Info} -> 
+	  %Scans the info read from the tcp connection
+	  {ok, Scanned, _} = erl_scan:string(binary_to_list(Info)),
+	  
+	  %Turns the scanned info into erlang format
+	  case erl_parse:parse_term(Scanned ++ [{dot,0}]) of 
+	    
+		%If the message is the Diagram id and the atom next_message
+		{ok, {N, next_message}} -> 
+		    %Checks if the diagram exists
+			case find_diagram(N, Diagrams) of
+			  not_created -> rip;
+			  Pid         -> 
+				%Makes the server simulate the next message
+				Pid ! {next_message, self()},
+			    %Confirmation that the diagram coordinator received the message
+				receive
+			      ok -> ok
+			    end,
+				%Receives the result from the diagram coordinator and sends it to the client
+			    receive
+		          {message_sent, From, To, Message, Message_number} -> 
+				    %Turns the result into binary
+					Temp = io_lib:format("~p", [{From, To, Message, Message_number}]) ++ "~",
```

Comment:
Why is this red?

Reply:
Github makes it red for some unknown reason, I can garantee you that it isnt breaking the code